sdks/RepoToolset/tools/Performance.targets

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <Target Name="PerformanceTest" Condition="'$(IsPerformanceTestProject)' == 'true'">
```

Comment:
><Target Name="PerformanceTest"  [](start = 2, length = 31)

The target will need to be defined in XUnit.targets as well.
See [https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/XUnit.targets#L21](https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/XUnit.targets#L21)

Reply:
Done...

Reply:
The target has to be defined for all projects, so that we can run `msbuild /t:PerformanceTest Foo.sln`. For non-perf-test project the target needs to be no-op.

---
In reply to: [167685658](https://github.com/dotnet/roslyn-tools/pull/158#discussion_r167685658) [](ancestors = 167685658)